sdk/rust/src/processor/mod.rs

```diff
@@ -225,6 +226,34 @@ impl<'a> TransactionProcessor<'a> {
                                         }
                                     };
                             },
+                            Message_MessageType::PING_REQUEST => {
+                                info!("sending PingResponse");
+                                let response = PingResponse::new();
+                                let serialized = match response.write_to_bytes() {
+                                    Ok(serialized) => serialized,
+                                    Err(err) => {
+                                        error!("Serialization failed: {}", err.description());
+                                        continue
+                                    }
+                                };
+                                let x : &[u8] = &serialized;
+                                match sender.reply(
```

Comment:
Have we determined if we can use `rustfmt` yet?  Not wanting to ding this line based on formatting without a tool, but I'm registering my opinion that I'm not a fan of the way this line is wrapped.

Reply:
Opinion registered. I do not believe that we can run rustfmt yet but that is something I can look into shortly.